packages/core/admin-client/src/authProviders/ProviderConfigs.ts

```diff
@@ -85,5 +100,7 @@ export {
   UserpassProviderConfig, 
   UserpassProvider, 
   CustomProviderConfig, 
-  CustomProvider
+  CustomProvider,
+  FunctionProviderConfig,
```

Comment:
[nit] I think in other places we are calling this `CustomFunction` not sure if its worth making the change here as well?